update.coffee

```diff
@@ -31,7 +31,7 @@ Promise.settle([propertiesPromise, propertyDescriptionsPromise]).then (results)
     properties[propertyName] = metadata
     console.warn "No description for property #{propertyName}" unless propertyDescriptions[propertyName]?
 
-  for propertyName, d of propertiesRaw
+  for propertyName of propertiesRaw
```

Comment:
In this case, I think not. propertyName is a key of array, d possibly meant description as value.

Here's a handy trick to distinguish between the two. http://stackoverflow.com/questions/17886135/mnemonic-for-remembering-coffeescripts-in-vs-of